mycroft/skills/intent_service.py

```diff
@@ -540,9 +341,9 @@ def handle_get_intent(self, message):
         intent = self._adapt_intent_match([utterance], [norm], lang)
```

Comment:
This method got removed, presuming it should now be `self.adapt_service.match_intent`

Reply:
also utterances need to match the new signature

Reply:
Should be handled now

Reply:
I could easily be wrong on this, but in the changed code:
`combined = [utterance + norm] if utterance != norm else [utterance]`
aren't `utterance` and `norm` strings? This will concatenate the strings as the first element of the list right?

Reply:
You are indeed correct, should be `[utterance, norm]`, fixing